src/main/webapp/app/shared/sidebar/sidebar-card-medium/sidebar-card-medium.component.scss

```diff
@@ -0,0 +1,14 @@
+.bg-module {
```

Comment:
I would suggest to do a follow-up PR to combine the classes we need for all SidebarCard sizes into a separate SCSS file. This way, we won't have to duplicate them.